src/dev/tslint/rules/noImportZonesRule.js

```diff
@@ -0,0 +1,208 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* @notice
```

Comment:
What should we do when use software licensed under MIT? Included a note as per https://github.com/elastic/kibana/blob/master/webpackShims/childnode-remove-polyfill.js

Reply:
This is how we comply with `The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.`, but you should include `/* eslint-disable @kbn/license-header/require-license-header */` too, since this isn't Apache 2.0 code.

Reply:
> since this isn't Apache 2.0 code.
you mean only this file is not Apache 2.0 anymore because of MIT license? or it's not needed to be licensed under Apache 2.0 at all?

Reply:
> you mean only this file is not Apache 2.0 anymore because of MIT license?
correct